access/types/forms.py

```diff
@@ -475,6 +487,19 @@ def grade_field(self, i, configuration):
         else:
             raise ConfigError("Unknown field type for grading: %s" % (t))
 
+        if self.exercise.get("feedback"):
+            # The partial_points is not meant to be used in feedback questionnaires.
+            # However if they both exist the grader should not crash.
+            ok = 1
```

Comment:
If `ok` becomes `1` here (if the exercise is a feedback questionnaire), then does this method return `1` for `ok`? The caller expects the value to be a boolean, so can't you keep it boolean? You can prevent the partial points scaling for feedback exercises in other ways too.